js/column.js

```diff
@@ -2212,6 +2212,66 @@ FacetColumn.prototype = {
         return this._comment;
     },
 
+    get hideNumOccurrences() {
+        if (this._hideNumOccurrences === undefined) {
+            this._hideNumOccurrences = (this._facetObject.hide_num_occurrences === true);
+        }
+        return this._hideNumOccurrences;
+    },
+
+    /**
+     * Returns the sortColumns when we're sorting this facet in scalar mode
+     * - uses row_order if defined.
+     * - otherwise it will be descending of num_occurrences and column order of base column.
+     * @type {Array}
+     */
+    get sortColumns() {
```

Comment:
Maybe the name of this function should be changed.. It makes me think it's for a "set of columns" rather than the "columns used to sort the current column".

Reply:
I can remove these APIs (sortColumns and hideNumOccurrences) now that I have added `scalarValuesReference`. I created these two at first, then I decided to add the third API. That's the only place that these two APIs are going to be used and it's internal.

But the name `sortColumns` is what I have used for other types of column, I can change that if you think it's not clear. What is a better name for it?